python/ray/dashboard/modules/metrics/dashboards/default_dashboard_panels.py

```diff
@@ -380,6 +380,58 @@ def max_plus_pending(max_resource, pending_resource):
             ),
         ],
     ),
+    Panel(
+        id=50,
+        title="Node TPU Tensorcore Utilization (Percentage)",
+        description="Percentage of tensorcore utilization for the TPUs on this node. Computed by dividing the number of tensorcore operations by the maximum supported number of operations during the sample period.",
+        unit="%",
+        targets=[
+            Target(
+                expr='sum(ray_tpu_tensorcore_utilization{{instance=~"$Instance",{global_filters}}}) by (instance, TpuIndex, TpuDeviceName)',
+                legend="Tensorcore Utilization: {{instance}}, tpu.{{TpuIndex}}, {{TpuType}}, {{TpuTopology}}",
```

Comment:
TpuToplogy won't show up in the label unless it's also part of the `sum by` expression.

Reply:
Fixed.